deployment/bentoml/mednist_classifier_bentoml.ipynb

```diff
@@ -94,6 +94,12 @@
     }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from bentoml.utils import cached_property  # noqa: F401
</pre><p>All the imports that were added here need to be moved back to the writefile cell they came from originally since we're writing out a source file to use with the bentoml tool.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1618/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>